crates/vertigo/src/computed/dependencies/graph.rs

```diff
@@ -116,20 +107,22 @@ impl Graph {
         }
     }
 
-    pub fn get_edges_to_refresh(&self, edges: &BTreeSet<GraphId>) -> Vec<GraphValueRefresh> {
+    pub fn has_listeners(&self, parent_id: &GraphId) -> bool {
+        self.parent_childs.relation_len(&parent_id) > 0
+    }
 
+    pub fn get_for_dropped(&mut self) -> Vec<GraphId> {
```

Comment:
`drain_removables` would be a better suited name, as we drain the source empty

Reply:
Done